systemtests/src/test/java/io/enmasse/systemtest/clients/BasicClientTest.java

```diff
@@ -124,27 +103,18 @@ public void topicSubscribeTest() throws Exception {
         assertEquals(10, receiver.getMessages().size());
     }
 
-    @Test
-    public void messageBrowseTest() throws Exception {
+    protected void doMessageBrowseTest(AbstractClient sender, AbstractClient receiver_browse, AbstractClient receiver_receive)
+            throws Exception {
         AddressSpace addressSpace = new AddressSpace("brokered-message-browse",
```

Comment:
Maybe should be better to do only one set up of address-space and addresses for all clients? (I don't know)
advantages/disadvantages:
We don't want to have garbage in queues/topic for run of next client, but it's time consuming to create everything again. Maybe we should use the same address space, but addresses(queue/topic) should be recreated?

Reply:
For now we will keep this implementation.